docs/how-to/working-with-rocfft.rst

```diff
@@ -353,8 +356,8 @@ The workflow for fields is as follows:
    added to the field (via calls to :cpp:func:`rocfft_field_add_brick`), and
    must point to memory on the device that was specified at that time.
 
-   For in-place transforms, only pass input pointers and do not pass output
-   pointers.
+   For in-place transforms, only pass input pointers and do not pass
```

Comment:
I would turn this into an important note:

.. important::
   For in-place transforms, only pass input pointers and do not pass...

Reply:
I've noticed a mis-wording in this paragraph and rewritten it.  There's a couple of other places where we could restructure this kind of text into notes so I can look at that in develop.